daffodil-test/src/test/scala/org/apache/daffodil/section05/simple_types/TestSimpleTypes.scala

```diff
@@ -421,6 +421,10 @@ class TestSimpleTypes {
   @Test def test_int_binary_04() { runner.runOneTest("int_binary_04") }
   @Test def test_int_binary_05() { runner.runOneTest("int_binary_05") }
 
+  @Test def test_binaryInt_unparseError() { runner.runOneTest("binaryInt_unparseError") }
+  @Test def test_hexBinary_unparseError() { runner.runOneTest("hexBinary_unparseError") }
+  @Test def test_calendar_unparseError() { runner.runOneTest("calendar_unparseError") }
+
```

Comment:
Suggest adding similar tests for all of our priitive types? That would be good confirmation that DAFFODIL-1681 is resolved, in addition to the original DAFFODIL-2124 (make  sure to update the commit message to reference both tickets).

Reply:
Done